.travis.yml

```diff
@@ -115,6 +115,14 @@ install:
   - python setup.py build_ext --inplace
   - python setup.py develop
 
+before_script:
```

Comment:
Can this whole thing go into something like a lint.sh file?  I'd like to be able to a) run it locally and b) make it increasingly strict over time as incremental cleanup occurs.

Reply:
Sure that is doable.